docs/reference/api/docker_remote_api_v1.22.md

```diff
@@ -392,6 +394,7 @@ Return low-level information on the container `id`
 		"HostConfig": {
 			"Binds": null,
 			"BlkioWeight": 0,
+            "BlkioWeightDevice": [{}],
```

Comment:
Not your fault, but it looks like the rest of this example uses tabs for indentation :disappointed:

Can you change to tabs to match the other lines? (just for this instance, because we normally use spaces) :relaxed: